src/scripts/farming/mutation/mutationTypes/EvolveNearBerryMinMutation.ts

```diff
@@ -0,0 +1,55 @@
+/// <reference path="./EvolveNearMutation.ts" />
+/**
+ * Mutation that requires at least some amount of specific Berry plants near an empty plot.
+ * The required Berry plants must be in the Berry stage for mutations to occur.
+ */
+class EvolveNearBerryMinMutation extends EvolveNearMutation {
+
+    berryReqs: StrictBerryReq;
+
+    constructor(mutationChance: number, mutatedBerry: BerryType, originalBerry: BerryType, berryReqs: StrictBerryReq, options?: MutationOptions) {
+        super(mutationChance, mutatedBerry, originalBerry, options);
+        this.berryReqs = berryReqs;
+    }
+
+    /**
+     * Determines if the plots near fit the requirements
+     * @param plots The list of nearby plots
+     */
+    nearPlotsFitRequirements(plots: number[]) {
+        const currentReqs: StrictBerryReq = {};
+
+        plots.forEach((idx) => {
+            const plot = App.game.farming.plotList[idx];
+            if (!plot.isUnlocked) {
+                return;
+            }
+            if (plot.isEmpty()) {
+                return;
+            }
+            if (plot.stage() !== PlotStage.Berry) {
+                return;
+            }
```

Comment:
Do we need all the extra checks in here?
If it's not unlocked or is empty, then this should also be true:
```suggestion
            if (plot.stage() !== PlotStage.Berry) {
                return;
            }
```

Reply:
I have a lot of sanity checks all over the place. Feel free to remove them.